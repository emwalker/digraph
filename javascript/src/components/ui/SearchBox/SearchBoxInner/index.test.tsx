import React from 'react'
import { shallow } from 'enzyme'

import FilterInput from './index'

const props = {
  onEnter: jest.fn(),
  location: {
    pathname: '/',
  },
  router: {
    push: () => {},
  },
  view: {
    queryInfo: {},
  },
}

describe('<FilterInput />', () => {
  // @ts-expect-error
  const wrapper = shallow(<FilterInput {...props} />)

  it('renders', () => {
    expect(wrapper).toMatchSnapshot()
  })
})
