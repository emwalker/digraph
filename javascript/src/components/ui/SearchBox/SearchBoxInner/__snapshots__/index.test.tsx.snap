// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<FilterInput /> renders 1`] = `
<form
  className="searchBox input-group"
  onSubmit={[Function]}
>
  <TextInput
    handleReturn={[Function]}
    queryInfo={Object {}}
  />
  <div
    className="input-group-button buttonContainer"
  >
    <button
      className="btn button"
      type="button"
    >
      Everything
    </button>
  </div>
</form>
`;
